eventmesh-connectors/eventmesh-connector-jdbc/build.gradle

```diff
@@ -37,12 +37,12 @@ dependencies {
     antlr("org.antlr:antlr4:4.13.0")
     implementation 'org.antlr:antlr4-runtime:4.13.0'
     implementation 'com.alibaba:druid:1.2.20'
-    implementation 'org.hibernate:hibernate-core:5.6.15.Final'
+    compileOnly 'org.hibernate:hibernate-core:5.6.15.Final'
     implementation project(":eventmesh-common")
     implementation project(":eventmesh-openconnect:eventmesh-openconnect-java")
     implementation project(":eventmesh-spi")
     implementation 'com.zendesk:mysql-binlog-connector-java:0.28.0'
-    implementation 'mysql:mysql-connector-java:8.0.32'
+    compileOnly 'mysql:mysql-connector-java:8.0.32'
 
```

Comment:
`mysql-connector-java` has been redirected to `mysql-connector-j`.

Although switching to `compileOnly` scope, its artifact jar package will still be bundled in our binary distribution. Therefore we will have to remove it manually before releasing, inform users to download its jar package, and put the jar file into the `lib` directory.

Reply:
This conversation is a inform and no changes are requested.

Reply:
Sorry I mistook it as `runtimeOnly`. It do remove artifacts from classpath, and no more manual operation is needed. Users will still have to download and put jar file by themselves.